pipelines/jobs/pipeline_job_template.groovy

```diff
@@ -62,6 +63,7 @@ pipelineJob("${BUILD_FOLDER}/${JOB_NAME}") {
     parameters {
         textParam('targetConfigurations', JsonOutput.prettyPrint(JsonOutput.toJson(targetConfigurations)))
         stringParam('activeNodeTimeout', "0", 'Number of minutes we will wait for a label-matching node to become active.')
+        stringParam('jdkVersion', jdkVersion, 'The JDK version of the pipeline e.g (11, 8, 17).')
```

Comment:
If the pipeline is a U version, won't we need to enter u here? `11u` for example? If that's the case, the description could probably be expanded to include that

Reply:
nope, I've handled it with an if statement here https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/f273f9a86004a0a0d1733baf58c5a5fe8e32c7b1/pipelines/build/openjdk_pipeline.groovy#L84-L91